.eslintrc.json

```diff
@@ -0,0 +1,44 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends":"airbnb",
+    "parser": "babel-eslint",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+        "jsx": false,
+        "experimentalObjectRestSpread": true,
+        "modules": false
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "no-console": 0,
```

Comment:
I think this rule should not be ignored. It should be 1, not 0

Reply:
yeah, thank you